### DISCUSSION TITLE
Gosund-SP111 Energieanzeige anpassen

### DISCUSSION THREAD

**Discussion by Firehowk** [camueller/SmartApplianceEnabler#251] [Q&A]
Hallo zusammen,

ich habe mehrere Gosund im Einsatz. Ein Teil konnte ich beim SAE noch nicht einbinden weil die Energieanzeige nur 3 Stellen nach dem Komma hat. Wie stellt man das auf 5 Stellen nach dem Komme um?

Gruss

Firehowk

**Comment by camueller** [comment]
Steht auf der Tasmota-Seite in der Doku: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Tasmota_DE.md
